fiftyone/plugins/utils.py

```diff
@@ -245,3 +245,8 @@ def _do_get_plugin_info(task):
             spec = gh_repo
 
         logger.debug("Failed to retrieve plugin info for %s: %s", spec, e)
+
+
+def get_builtin_plugins_dir():
+    """Returns the path to the built-in plugins directory."""
```

Comment:
Why built-in vs builtin you may ask... just following the norms from python itself:
<img width="291" alt="image" src="https://github.com/user-attachments/assets/ee5dbb63-c803-435c-a892-4107f2442b83" />